File: app/src/main/java/com/websarva/wings/android/kaigonote/MosiokuriActivity.java
package com.websarva.wings.android.kaigonote;

import android.os.Bundle;
import android.view.View;

import androidx.appcompat.app.AppCompatActivity;

import com.websarva.wings.android.kaigonote.databinding.MosiokuriBinding;

public class MosiokuriActivity extends AppCompatActivity {
    private MosiokuriBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = MosiokuriBinding.inflate(getLayoutInflater());
        View view = binding.getRoot();
        setContentView(view);
    }
}

File: app/src/main/java/com/websarva/wings/android/kaigonote/MosiokuriActivity.kt
package com.websarva.wings.android.kaigonote

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.websarva.wings.android.kaigonote.databinding.MosiokuriBinding

class MosiokuriActivity : AppCompatActivity() {
    private var binding: MosiokuriBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = MosiokuriBinding.inflate(layoutInflater)
        val view: View = binding!!.root
        setContentView(view)
    }
}